src/actions/helpers/index.ts

```diff
@@ -42,3 +65,32 @@ export const validateOwnerAndStatus = async (
   await validateUserOwnsDomain(domainId, potentialOwner, registrar);
   await validateStatus(domainId, registrar, desiredLock, potentialOwner);
 };
+
+/**
+ * Get a token's price using the CoinGecko API.
+ * If it can't be found on CoinGecko, that token's derived ETH ratio
+ * given by a DEX subgraph is used instead
+ * @param paymentTokenAddress The contract address of the ERC20 payment token
+ * @param derivedETH The derived ETH ratio
+ * @returns The price of the token in USD
+ */
+export const getTokenPrice = async (
+  paymentTokenAddress: string,
+  derivedETH: string
```

Comment:
It seems like `derivedETH` is optional in this function if `paymentTokenAddress` is available via coingecko, thus `derivedETH` should be an optional parameter.